components/dependencies.mdx

```diff
@@ -1,59 +1,105 @@
-| url      | The fully composed URL of the reference interface. This will take the format, `<protocol>://<username>:<password>@<host>:<port><path>`.                                          |
-| protocol | The protocol of the interface being referenced. This will always be the specific value assigned to the interface by the developer.                                               |
-| username | The username of the interface being referenced. Not all interfaces have usernames, so this will be an empty string if none is set.                                               |
-| password | The password of the interface being referenced. Not all interfaces have passwords, so this will be an empty string if none is set.                                               |
-| host     | The host value of the interface being referenced. This value is usually dynamic to accomodate the differences between service discovery solutions available to each environment. |
-| port     | The port value of the interface being referenced.                                                                                                                                |
-| path     | The path value of the interface being referenced. Not all interfaces have paths, so this will be an empty string if none is set.                                                 |
+<Card title="SPAs and single page apps" icon="link" href="/architecture/static-websites">
+  SPAs and static websites run in the browser and need to connect to the external **ingress** URLs. Click here to learn more.
+</Card>
+
+Finally, let's deploy our application! We haven't written any code to connect to the backend API, but you'll see that the
+backend got automatically deployed, and you're welcome to go try writing features that leverage it. 
+
+## Available pointers
+
+Service addresses aren't the only thing you can inject into your own component from dependencies. Developers can
+point to just about any resoure declared by the dependency to enrich their own configuration:
+
+| Key                             | Description                                        |
+| ------------------------------- | -------------------------------------------------- |
+| `dependencies.<name>.services`  | Allows you to inject values associated with [services](/components/services) inside the dependency  |
+| `dependencies.<name>.databases` | Allows you to inject values associated with [databases](/components/databases) inside the dependency |
+| `dependencies.<name>.outputs`   | Allows you to inject output values issued by the dependency |
+
+## Debugging dependencies
+
+By default Architect will pull dependencies down from the cloud registry on every deployment. But if you want to test changes to,
+two components at once, you can do so by linking the dependency to your host machine with `architect link`:
```

Comment:
```suggestion
two components at once, you can do so by linking the dependency to your host machine with `architect link`. This will cause Architect to use the local version of the components rather than trying to pull them from the Architect cloud registry. This is helpful when you are actively developing multiple components that are dependent on one another. 
```